### PR TITLE
doc: add long about messages for subcommands

### LINKED ISSUE #2258: "deno help fetch" should have better documentation
```
> deno help fetch
deno-fetch
Fetch the dependencies

USAGE:
    deno fetch <file>

FLAGS:
    -h, --help    Prints help information

ARGS:
    <file>
```
This is not at all clear what it does. It would be great if it had an example like 
```
deno fetch https://deno.land/std/http/file_server.ts
```
An "file" should probably be "url" instead

### PR DESCRIPTION
**Pull request by kevinkassimo**
Closes #2258.

Type deno <subcommand> -h to view descriptions.

### CODE REVIEW

File: cli/flags.rs [hunk 1]
```diff
@@ -103,31 +103,90 @@ pub fn create_cli_app<'a, 'b>() -> App<'a, 'b> {
...
+",
+        ),
     ).subcommand(
       SubCommand::with_name("fetch")
         .setting(AppSettings::DisableVersion)
         .about("Fetch the dependencies")
-        .arg(Arg::with_name("file").takes_value(true).required(true)),
+        .long_about(
+          "
+Fetch remote dependencies recursively.
```
Comment (bartlomieju) [inline cli/flags.rs]: `fetch` compiles downloaded deps as well, I guess it's worth noting

Comment (kevinkassimo) [reply cli/flags.rs]: Agreed. I'll update this once I get more feedback on this (and after the pressure on travis has been alleviated)

File: cli/flags.rs [hunk 2]
```diff
@@ -103,31 +103,90 @@ pub fn create_cli_app<'a, 'b>() -> App<'a, 'b> {
...
-        .about("Print the version"),
+        .about("Print the version")
+        .long_about(
+          "
+Print current version of Deno.
+
+Includes versions of Deno, V8 JavaScript Engine, and the TypeScript
+compiler.
+
+  deno version
```
Comment (ry) [inline cli/flags.rs]: The example is superfluous

File: cli/flags.rs [hunk 3]
```diff
@@ -103,31 +103,90 @@ pub fn create_cli_app<'a, 'b>() -> App<'a, 'b> {
...
     ).subcommand(
       SubCommand::with_name("eval")
         .setting(AppSettings::DisableVersion)
         .about("Eval script")
-        .arg(Arg::with_name("code").takes_value(true).required(true)),
+        .long_about(
+          "
+Evaluate provided script.
+
+  deno eval 'console.log(\"hello world\")'
```
Comment (ry) [inline cli/flags.rs]: you can avoid the backslashes if you use double quotes on the outside and single quotes on the inside

Comment (kevinkassimo) [reply cli/flags.rs]: There is another pair of double quotes outside, so I believe backslashes are inevitable either way...

**Pull request merged**